python/sanlock.c

```diff
@@ -178,6 +181,17 @@ add_align_flag(long align, uint32_t *flags)
     return 0;
 }
 
+static void
+__set_value_error(const char* msg, PyObject* obj)
```

Comment:
"msg" is better named "fmt", since it is a format string, and it should contain a placeholder for obj.